src/deploy/functions/containerCleaner.ts

```diff
@@ -128,6 +133,55 @@ export class ContainerRegistryCleaner {
   }
 }
 
+// top level function for purging artifacts
```

Comment:
Prefer describing the behavior in a little more detail , and documenting any non-obvious behavior (like what happens if location is omitted). Something like:
```
/**
* Deletes all artifacts from GCF folder in GCR.
* @param projectId
* @param location: the subdomain to clean up. If omitted, will delete all locations.
*/
```

Reply:
Added TSDoc comments to all functions